README.md

```diff
@@ -211,6 +211,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--output` - Send the output results to a file
     * `--format` [default: text] - Change the output format (e.g. json)
     * `--commit-uuid` [default: latest commit of current git branch] - Set the commit UUID that will receive the results on Codacy
+    * `--skip-commit-uuid-validation` | -s  Skip commit UUID validation
```

Comment:
Fix the formatting for the new flag:

```suggestion
    * `--skip-commit-uuid-validation` [default: false] - Skip commit UUID validation
```

(I propose that we don't include the short flag name to make it consistent with all the others.